### PR TITLE
Volunteers Page : Added Header Image & Volunteer Cards

### PR DESCRIPTION
**Pull request by jkrlr**
#28 

# Proposed Changes
- Added Header image 
- Added Cards of Volunteer info

# Additional Info
- Styling for `vol__header` div & `Vol__Cards` div haven't properly done yet for large as well as small size screens.

# Screenshots

- Header img section looks like this.

Screenshot from 2021-04-20 09-59-39

- ### About Card info
  - Cards looks like as shown below.
  - Designation(Hindi, Mathematics in my case) has been putted just below the img.
  - 2nd card of first row shows hover effect of cards.
  - if designation is none, then the card will be look 3rd card of first row.
  
Screenshot from 2021-04-20 10-00-01

### CODE REVIEW

File: apps/volunteers/models.py [hunk 1]
```diff
@@ -46,7 +47,10 @@ def save(self, *args, **kwargs):
         self.active = (self.active is True)
         super(Volunteer, self).save(*args, **kwargs)
 
-
+    def get_absolute_url(self):
+        """Returns the url to access a particular volunteer instance."""
+        return reverse('volunteers:profile', args=[str(self.id)])    
+
```
Comment (jkrlr) [inline apps/volunteers/models.py:53]: `get_absolute_url` function for accessing the particular volunteer instance.

File: templates/volunteers/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{% extends 'sidebar.html' %} 
+{% load static %} 
+
+{% block title %} Jagrati | Volunteers {% endblock title %} 
+
+{% block style %}
+<link rel="stylesheet" type="text/css" href=" {% static 'volunteers/css/style.css' %}" />
+{% endblock style %}
```
Comment (jkrlr) [inline templates/volunteers/index.html:8]: Added stylesheet in separate folder.

File: apps/volunteers/models.py (hunk 1 above)
Comment (garg3133) [reply apps/volunteers/models.py:53]: This is cool.

**Comment by garg3133** [comment]
Great work @jkrlr! It looks really good (except for the bg and shadow of cards, which looks odd). Thank you for working on this. I guess I'll have to complete this page from here.

**Comment by jkrlr** [comment]
> Great work @jkrlr! It looks really good (except for the bg and shadow of cards, which looks odd). Thank you for working on this. I guess I'll have to complete this page from here.

Sure @garg3133 sir! You can.

Actually, orange color theme looks very old to me. So, I tried girlscript projects theme(different from our main theme) in card component. 
I've a small suggestion for you, "In global css file put main color theme, font-family etc in form of css variables. So that one can use that variables in their css files while building new parts of our project. Because, In future if we want to change our color theme. Then, To do that we've to make changes in only global files."

**Pull request closed without merging**